JoinColony/colonyCDapp
# feat: added meatball menu in completed action

## Description
Added meatball menu to completed action. For now, it is only added to these actions because adv payments are not ready yet:
- Simple Payment
- Transfer funds
- Mint tokens
- Manage Reputation
- Manage Permissions

## Testing

* Create a action listed above
* Use meatball menu options to redo action or check on explorer
<img width="754" alt="image" src="https://github.com/JoinColony/colonyCDapp/assets/76940796/1c4bca8d-8d92-44f7-868b-55f7a890e1e5">


## Diffs

**New stuff** ✨

* New stuff goes here

## TODO

- Implement this functionality in Payment Builder, Split Payment, Staged Payment, Streaming Payment.

Resolves [#2311](https://github.com/JoinColony/colonyCDapp/issues/2311)
